pinot-core/src/main/java/org/apache/pinot/core/data/manager/realtime/HLRealtimeSegmentDataManager.java

```diff
@@ -287,8 +287,7 @@ public void run() {
               new RealtimeSegmentConverter(_realtimeSegment, tempSegmentFolder.getAbsolutePath(), schema,
                   _tableNameWithType, tableConfig, realtimeSegmentZKMetadata.getSegmentName(), _sortedColumn,
                   HLRealtimeSegmentDataManager.this._invertedIndexColumns, _noDictionaryColumns,
```

Comment:
(nit) Remove `HLRealtimeSegmentDataManager.this.`

Reply:
done